core/config-manager/configurationmanager_test.go

```diff
@@ -254,6 +246,34 @@ ssl:
   certPwdFile:
 
 commonkey3 : filesource
+`)
+
+	loadbalanceConf := []byte(`
+--- 
+cse: 
+  loadbalance: 
+    backoff: 
+      backoff: 
+        maxMs: 400
+        minMs: 200
+        kind: constant
+      retryEnabled: true
+      retryOnNext: 2
+      retryOnSame: 3
+      serverListFilters: zoneaware
+      strategy: 
+        name: WeightedResponseForTargetService
+    backoff: 
```

Comment:
same as 255, I think it is duplicated

Reply:
That i have given intentionly to verify that the string can be anything even it can be the tags which are already there